docs/src/pages/[platform]/connected-components/authenticator/customization/override-function-calls/example.angular.mdx

```diff
@@ -0,0 +1,107 @@
+import { Tabs, Alert } from '@aws-amplify/ui-react';
+import { CURRENT_MAJOR_VERSIONS, PREV_MAJOR_VERSIONS } from '@/data/frameworks';
+
+export let latestVersion = CURRENT_MAJOR_VERSIONS['angular'];
+export let prevVersion = PREV_MAJOR_VERSIONS['angular'];
+
+_app.component.html_
+
+```html{1}
+<amplify-authenticator
+  [services]="services"
+  initialState="signUp"
+>
+  <ng-template
+    amplifySlot="authenticated"
+    let-user="user"
+    let-signOut="signOut"
+  >
+    <h2>Welcome, {{ user.username }}!</h2>
+    <button (click)="signOut()">Sign Out</button>
+  </ng-template>
+</amplify-authenticator>
+
+```
+
+_app.component.ts_
+<Tabs.Container defaultValue={latestVersion}>
+  <Tabs.List>
+    <Tabs.Item value={latestVersion}>ui-angular v{latestVersion} (latest)</Tabs.Item>
+    <Tabs.Item value={prevVersion}>ui-angular v{prevVersion}</Tabs.Item>
```

Comment:
Think we need to use the full package name here or just the version (since the reader is already on the framework page. Same feedback if we are doing this elsewhere